segmentstore/server/src/test/java/io/pravega/segmentstore/server/tables/TableEntryHelpers.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.server.tables;
+
+import io.pravega.common.util.HashedArray;
+import io.pravega.segmentstore.contracts.tables.TableEntry;
+import io.pravega.segmentstore.contracts.tables.TableKey;
+
+/**
+ * Helper methods for testing with {@link TableEntry} instances.
+ */
+final class TableEntryHelpers {
+
+    static boolean areEqual(TableEntry e1, TableEntry e2) {
```

Comment:
Why can't you do this with `equals`?

Reply:
Because this was test-only so I didn't want to modify the original classes. But now that you mention it, I think it makes sense.